src/lib/viewers/controls/annotations/annotations-button-region-tooltips/AnnotationsButtonRegionTooltips.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext, PropsWithChildren } from 'react';
+
+import { ExperiencesContext } from '../../experiences';
+
+import AnnotationsFlowTooltip from './AnnotationsFlowTooltip';
+import PersistentOnboardingBoxEditAnnotationsTooltip from './PersistentOnboardingBoxEditAnnotationsTooltip';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function AnnotationsButtonRegionTooltips({ children, isEnabled }: Props): JSX.Element {
+    const { experiences } = useContext(ExperiencesContext);
+
+    const { persistentOnboardingBoxEditAnnotations: experience } = experiences;
+    const canShow = experience ? experience.canShow : false;
```

Comment:
Only because `experience` is possibly `undefined`. I assumed that when `box-content-preview` will be used outside of `EndUserApp` contexts, `previewExperiences` will be `undefined`, so I have to guard against that.

I saw 2 possible approaches:
- Define the `Experiences` type with possibly `undefined` properties and then guard against that in the components using them
- Define the `Experiences` type with required properties and declare some mock experiences at `ExperiencesProvider` level

I chose the first path. Let me know if you think this would be handled better somehow differently.